### ISSUE TITLE
Duplicate deposit log import

### ISSUE THREAD

**Issue by JustinDrake** [sigp/lighthouse#1995]
## Description

I was surprised to see the following logs on my Lighthouse beacon node:

```
Nov 27 23:40:35.774 INFO Waiting for genesis                     wait_time: 3 days 12 hrs, peers: 53, service: slot_notifier
Nov 27 23:40:43.580 INFO Imported deposit log(s)                 new: 1, total: 24900, latest_block: 11341971, service: eth1_rpc
Nov 27 23:40:47.782 INFO Waiting for genesis                     wait_time: 3 days 12 hrs, peers: 50, service: slot_notifier
Nov 27 23:40:50.388 INFO Imported deposit log(s)                 new: 1, total: 24900, latest_block: 11341971, service: eth1_rpc
```

The above seems to suggest duplicate work was done in importing deposit logs.

## Version

v1.0.1

**Comment by pawanjay176** [comment]
Similar to https://github.com/sigp/lighthouse/issues/2006#issuecomment-735704148

**Comment by pawanjay176** [comment]
Fixed in #2038
